config/production.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Should this file be gitignored?

Reply:
No. This file is for production settings. These ids in the settings are plainly used on pages and are not secret.